diagnostic.md

```diff
@@ -35,5 +40,7 @@ Record your responses inside the fenced code blocks below each question.
     pass it data?
 
     ```html
-    <!-- your response here -->
+      {{#each my-contact.contacts as |contact|}}
+        <li>{{contact.my-phone}}</li>
+      {{/each}}
```

Comment:
{{#each contact.phones as |phone|}} 
{{my-phone phone=phone}}
{{/each}}

Close, you have to load the component and bind.